bundler/lib/bundler/man/bundle-config.1.ronn

```diff
@@ -193,9 +127,10 @@ learn more about their operation in [bundle install(1)](bundle-install.1.html).
    Ignore the current machine's platform and install only `ruby` platform gems.
    As a result, gems with native extensions will be compiled from source.
 * `frozen` (`BUNDLE_FROZEN`):
-   Disallow changes to the `Gemfile`. When the `Gemfile` is changed and the
-   lockfile has not been updated, running Bundler commands will be blocked.
-   Defaults to `true` when `--deployment` is used.
+   Disallow any automatic changes to `Gemfile.lock`. Bundler commands will
+   be blocked unless the lockfile can be installed exactly as written.
```

Comment:
We could add a sentence like the following after this one to further clarify:

> Usually this will happen when changing the `Gemfile` manually and forgetting to update the lockfile through `bundle lock` or `bundle install`.